docs/conf/cincinnati/2018/news/welcome.rst

```diff
@@ -0,0 +1,63 @@
+
+Here's a quick overview of the schedule:
+
+* Day 1: Talks on all things technical documentation
+* Day 2: Unconference discussion and open space
+* Days 3-5: Documentation sprints and writing days
+
+Write the Docs Cincinnati + Open Help is scheduled for August 18-22, 2018 at `Union Hall <http://unionhallcincy.com/>`_ in the historic Over the Rhine neighborhood in `Cincinnati, Ohio <https://www.google.com/maps/place/Union+Hall/@39.109885,-84.515631,15z/data=!4m5!3m4!1s0x0:0x3caee5bfc65866b3!8m2!3d39.109885!4d-84.515631>`_.
+
+Call for Organizers and Volunteers
+----------------------------------
+
+We need help to make this event a success! If you are interested in volunteering or helping to organize, please email us at cincinnati@writethedocs.org. It's a great opportunity to get involved with the community and also receive a free ticket.
+
+Call for Proposals
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience. Whether you’ve been writing documentation for decades or you’ve just started this year, we want to hear from you!
+
+Speaking at Write the Docs is a fantastic way to share your ideas with our inspiring community of documentarians. If there is something you’d really like to see a talk on, submit a proposal on it, or refer someone else!
+
+Read our full `Call for Proposals <http://www.writethedocs.org/conf/cincinnati/2018/cfp/>`__ on the website.
+The Call for Proposals will be open until **Midnight EST June 18, 2018**.
+
+Website
+-------
+
+The `conference website <http://www.writethedocs.org/conf/cincinnati/2018/>`_ is up and running, with all the latest details about the event. 
+
+As with all things on the web, there may be a few lingering bugs with the design. Feel free to reach out to us by replying to the email, or emailing us at support@writethedocs.org if you see any issues.
+
+Tickets
+-------
+
+Tickets will officially go on sale in **May 2018**.
+
+In the meantime, you can see our 2018 ticket prices on our `Tickets <http://www.writethedocs.org/conf/cincinnati/2018/tickets/>`_ page. As with our other conferences, we have tiered pricing to make it fair for everyone. Please buy the ticket that best fits your situation.
+
+We keep ticket prices low so that it is accessible to the widest range of people possible. However, if you can't afford these prices and still wish to attend, please email us at cincinnati@writethedocs.org, and we can see how we can help.
+
+Stay Updated
+------------
+
+Want to find out what's happening with the conference, or enjoy our monthly global community newsletter?
```

Comment:
should we set up a separate segment in MailChimp for east coast? or just reuse the NA segment?